internal/hmyapi/backend.go

```diff
@@ -112,31 +112,31 @@ func GetAPIs(b Backend) []rpc.API {
 			Public:    true, // FIXME: change to false once IPC implemented
 		},
 		{
-			Namespace: "hmyv2",
+			Namespace: "hmy_v2",
 			Version:   "1.0",
 			Service:   apiv2.NewPublicHarmonyAPI(b),
 			Public:    true,
 		},
 		{
-			Namespace: "hmyv2",
+			Namespace: "hmy_v2",
```

Comment:
Same question like on other PR, will it break trustwallet or anyone

Reply:
Asked

Reply:
excellent, just read in tg. looks good, we should merge .